### PR TITLE
test post request

### PR DESCRIPTION
**Pull request by volodymyrss**

### CODE REVIEW

File: tests/test_server_basic.py [hunk 1]
```diff
@@ -520,10 +535,11 @@ def test_no_token(dispatcher_live_fixture, async_dispatcher):
 
 
 @pytest.mark.parametrize("selection", ["range", "280200470010.001"])
+@pytest.mark.parametrize("method", ['get', 'post'])
 @pytest.mark.dda
 @pytest.mark.isgri_plugin
 @pytest.mark.xfail
-def test_isgri_image_no_pointings(dispatcher_live_fixture, selection):
+def test_isgri_image_no_pointings(dispatcher_live_fixture, selection, method):
```
Comment (burnout87) [inline tests/test_server_basic.py:542]: This test will be moved in the dispatcher-plugin-integral repo.

File: tests/test_server_basic.py [hunk 2]
```diff
@@ -560,7 +577,8 @@ def test_isgri_image_no_pointings(dispatcher_live_fixture, selection):
 
 @pytest.mark.dda
 @pytest.mark.isgri_plugin
-def test_isgri_image_fixed_done(dispatcher_live_fixture):
+@pytest.mark.parametrize("method", ['get', 'post'])
+def test_isgri_image_fixed_done(dispatcher_live_fixture, method):
```
Comment (burnout87) [inline tests/test_server_basic.py:581]: Same for this one

File: tests/test_server_plugin_integral_all_sky.py [hunk 3]
```diff
@@ -1,3 +1,5 @@
+#TODO: this was here to make an example of dispatcher interface, but the interface is now fixed by other tests
+
```
Comment (burnout87) [inline tests/test_server_plugin_integral_all_sky.py:2]: Does this mean that is no longer necessary?

**Review by burnout87 — CHANGES_REQUESTED** [review]

File: tests/test_server_basic.py (hunk 1 above)
Comment (volodymyrss) [reply tests/test_server_basic.py:542]: yes!

File: tests/test_server_plugin_integral_all_sky.py (hunk 3 above)
Comment (volodymyrss) [reply tests/test_server_plugin_integral_all_sky.py:2]: it will not be necessary once moved

**Review by burnout87 — APPROVED** [review]

**Pull request merged**